app/actions/GroupActions.ts

```diff
@@ -43,6 +44,25 @@ export function removeMember(membership: Record<string, any>): Thunk<any> {
     },
   });
 }
+export function editMembership(
+  membership: MembershipType,
+  role: string
+): Thunk<any> {
+  return callAPI({
+    types: Membership.UPDATE,
+    endpoint: `/groups/${membership.abakusGroup}/memberships/${membership.id}/`,
+    method: 'PATCH',
+    body: {
+      role,
+    },
+    schema: membershipSchema,
+    meta: {
+      id: membership.id,
+      groupId: membership.abakusGroup,
```

Comment:
Are these used?

Reply:
Nope, I just blindly copied the `removeMember` function above 🙈